features/payment-flows-and-management/payouts/payout-link.md

```diff
@@ -0,0 +1,165 @@
+    "confirm": false,
+    "session_expiry":2592000
+}'
+```
+
+#### 3. Customizing a payout link during creation:
+
+You can also customize a specific payout link by including the **payout_link_config** object while creating a link during [payouts/create](https://api-reference.hyperswitch.io/api-reference/payouts/payouts--create) call as well. The UI is customizable during link creation. However, domain_name cannot be customized and is always read from a business profile's config.
+
+<pre class="language-markup"><code class="lang-markup">curl --location 'https://sandbox.hyperswitch.io/payouts/create' \
+--header 'Content-Type: application/json' \
+--header 'Accept: application/json' \
+--header 'api-key: YOUR_API_KEY' \
+--data '{
+    "amount": 1130,
+    "currency": "USD",
+    "confirm": false,
+    "customer_id": "cus_123",
+    "return_url": "https://hyperswitch.io",
+    "description": "For selling Tshirt",
+    "payout_link": true,
+    "billing": {
+        "address": {
+            "line1": "1467",
+            "line2": "Harrison Street",
+            "line3": "Harrison Street",
+            "city": "San Fransico",
+            "state": "CA",
+            "zip": "94122",
+            "country": "US",
+            "first_name": "John",
+            "last_name": "Doe"
+        },
+        "phone": {
+            "number": "8056594427",
+            "country_code": "+91"
+        }
+    },
+    "session_expiry": 2592000,
+<strong>    "payout_link_config"</strong>: {
+        "merchant_logo": "https://i.imgur.com/N18JAS2.jpg",
+        "color_scheme": {
+            "background_primary_color": "#36D399",
+            "sdk_theme": "#36D399"
+        }
+    }
+}'
+</code></pre>
+
+<figure><img src="../../../.gitbook/assets/payout_links.png" alt=""><figcaption></figcaption></figure>
+
+### FAQ
+
+<details>
+
+<summary>Can I create a payout link pointing to my custom domain?</summary>
+
+Yes. Your custom domain can be included in the default payout_link_config object as part of the business profile update.
+
+This involves adding CNAME records and TLS certificates which ends up being a slightly complex process. Please reach out to our [Support](https://join.slack.com/t/hyperswitch-io/shared_invite/zt-1k6cz4lee-SAJzhz6bjmpp4jZCDOtOIg) to test this feature out with your custom domain.
+
+</details>
+
+<details>
+<summary>What are the benefits of using a Payout link?</summary>
+Payout links simplify the process of sending money, eliminating the operational complexities of bank transfers or payouts. With just a few clicks, you can create a payout link. Once generated, we notify the recipient, who can redeem the money at their convenience.
+</details>
+
+</details>
+
+<details>
+
+<summary>How long is the Payout link valid for?</summary>
+
+The payout link is valid for 15 minutes by default. However you can increase the validity to upto 3 months (7890000) by passing the time in seconds in `session expiry` in the create payout link call
```

Comment:
> [...] by passing the time in seconds in `session expiry` in the create payout link call [...]

Try to match the field names in the request body as much as possible: `session_expiry` instead of `session expiry`.